pyenv/install.sh

```diff
@@ -31,6 +31,9 @@ function __init_pyenv() {
             echo '# added by Webi for pyenv' >> ~/.config/fish/config.fish
             echo 'pyenv init - | source' >> ~/.config/fish/config.fish
```

Comment:
These two existing lines need to be replaced entirely.

The two new lines need to be added verbatim.

```bash
echo 'status is-login; and pyenv init --path | source' >> ~/.config/fish/config.fish
echo 'status is-interactive; and pyenv init - | source' >> ~/.config/fish/config.fish
```

Reply:
Also, please `git rebase -i main` to squash your changes into a single commit. :)

Reply:
@coolaj86 the requested changes has been made, please review :)